catalog/dags/providers/provider_api_scripts/europeana.py

```diff
@@ -223,7 +243,50 @@ def get_batch_data(self, response_json: dict) -> None | list[dict]:
         return response_json.get("items")
 
     def get_record_data(self, data: dict) -> dict:
-        return self.record_builder.get_record_data(data)
+        return self.record_builder.get_record_data(
+            data,
+            self._get_additional_item_data(data),
```

Comment:
I hear that. I feel nervous about it, because the overall structure seems so flexible that I worry about things getting jumbled up that way. What would you think of something like `data | {"item webresource": webresource}`?

Reply:
Good point, I think caution is understandable! I like that idea.